src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let stringArray = '';
```

Comment:
Why do you use the word `array` in the title, but it is a string type?
Try to choose a more meaningful naming.